services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -49,6 +61,10 @@ interface DocumentTableProps {
   additionalColumnProps: { key: string; colProps: any }[];
   fileOperationPermissionMap: { operation: FileOperations; permission: string | boolean }[];
   userRoles: string[];
+  documentsCompression: ActionCreator<typeof documentsCompression>;
```

Comment:
Oooh actually if these _are_ generic functions that can be used by any document, we can just import them instead of passing them in as props. Way simpler to implement for the next one.

Reply:
This should be using the `mapDispatchToProps` approach now.